src/lightning_app/cli/lightning_cli.py

```diff
@@ -216,21 +217,24 @@ def cluster_logs(cluster_name: str, to_time: arrow.Arrow, from_time: arrow.Arrow
             f" Please select one of the following: [{', '.join(clusters.keys())}]"
         )
 
-    log_reader = _cluster_logs_reader(
-        client=client,
-        cluster_id=clusters[cluster_name],
-        start=from_time.int_timestamp,
-        end=to_time.int_timestamp,
-        limit=limit,
-        follow=follow,
-    )
+    try:
```

Comment:
Would be better to understand why this happens.

Reply:
What do you mean? What would be actionable to do here exactly?

Reply:
I meant it would be good to understand why `Error while reading logs` is triggered instead of catching it. Logs are key and it needs to be reliable.

Reply:
I guess the stack trace in debug mode.

Reply:
@tchaton ok

FYI lightning_cloud package uses `DEBUG` envvar, so  this means we're using 2 envvars in the same CLI

Reply:
Done; reusing value from lighting_cloud package to keep those things in sync, though it's being imported in the constants